onnxscript/evaluator.py

```diff
@@ -418,8 +442,3 @@ def default_as(temp_default: Evaluator):
         yield
     finally:
         set_default(old_default)
-
-
-def eval(schema, inputs, attributes):
```

Comment:
I removed this because I thought it duplicates with `evaluator.default().eval`. LMK if this `eval` is intended to be a public api?

Reply:
cc @gramalingam

Reply:
Looks fine either way.